src/tests/ftest/util/mdtest_utils.py

```diff
@@ -196,6 +198,7 @@ def get_launch_command(self, manager, attach_info, processes, hostfile):
             "CRT_ATTACH_INFO_PATH": attach_info,
             "MPI_LIB": "\"\"",
             "DAOS_SINGLETON_CLI": 1,
+            "D_LOG_FILE": client_log
```

Comment:
For now we don't have such test which I can think of in case of mdtest which will lead client_log==None. But if you know of any please let me know.
But to answer your question, theoretically it will export D_LOG_FILE=None.

Reply:
Ok.